app/helpers/exchanges_helper.rb

```diff
@@ -34,12 +33,13 @@ def last_viewed_page_path(exchange)
   end
 
   def post_page(post)
-    if controller.kind_of?(ExchangesController) && params[:action] == 'show' && @posts
-      # Speed tweak
+    # Speed optimization
+    if controller.is_a?(ExchangesController) &&
+       params[:action] == "show" &&
+       @posts
```

Comment:
Use 4 (not 3) spaces for indenting a condition in an `if` statement spanning multiple lines.